zarr/convenience.py

```diff
@@ -1277,7 +1277,9 @@ def open_consolidated(store: StoreLike, metadata_key=".zmetadata", mode="r+", **
     """
 
     # normalize parameters
-    store = normalize_store_arg(store, storage_options=kwargs.get("storage_options"), mode=mode)
+    zarr_version = kwargs.get('zarr_version', None)
```

Comment:
Perhaps more for a follow-up, but if this is getting extracted does it make sense to list as a parameter?

Reply:
I notice that's not happening with `storage_options` either, so it probably is more of a higher-level aesthetic decision.

Reply:
I am fine doing it either way.